### DISCUSSION TITLE
grottserver multiregister command

### DISCUSSION THREAD

**Discussion by hugoboss00** [johanmeijer/grott#594] [General]
how does the multiregister command work? (syntax of it)
maybe you can give an example for it?
I want to use it to set starttime/stoptime and enable of a battery first slot (Registers 1100,1101,1102). They need to be set in a single operation

**Comment by PedroKTFC** [comment]
[This](https://github.com/johanmeijer/grott/issues/538) might clarify things.

**Reply by hugoboss00** [reply]
Thanks, but sorry the link is dead
btw. i did a search for multiregister in the whole repo, but didnt found anything describing it

**Reply by PedroKTFC** [reply]
Rats! No idea why that went t!ts up. Hopefully fixed now!

**Reply by hugoboss00** [reply]
ok, that one i found already. but you implemented it on a different repo.
I want to stay at  @johanmeijer 's repo, also because multiregister command should be already there.
I am just missing the description of multiregister.

**Reply by PedroKTFC** [reply]
I don't think the multi-register has been implemented. That's one of the reasons I forked my version (plus the libscrc problem I had).

**Comment by johanmeijer** [comment]
Multiregister command has been implemented (see #191  ). During the rebuild I am working I detected that it is a little bit coded different then I expected. But you can try to use it. 

To be honest I never used it (yet). 

The requrement for libscrc will be removed in the next version of Grott.

**Reply by PedroKTFC** [reply]
Have to say I never got it to work.

**Reply by johanmeijer** [reply]
I will test it in combination with Grott 3.0

**Comment by hugoboss00** [comment]
Ok, thanks for clarification. I will take a look at @PedroKTFC 's implementation
